src/redux/books/books.js

```diff
@@ -15,10 +31,13 @@ export const addBook = ({ id, author, title }) => ({
   },
 });
 
-export const removeBook = (id) => ({
-  type: REMOVE_BOOK,
-  payload: id,
-});
+export const removeBook = (id) => {
+  console.log(id);
```

Comment:
- [ ] Following best practices for JavaScript, you should not commit console.log to your repo. It kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website.